tests/testthat/test-mixedmodelsglmm.R

```diff
@@ -5,20 +5,20 @@ context("Generalized Linear Mixed Models")
 {
   skip_on_os("mac")   # problems with precision outside of windows
   skip_on_os("linux") # problems with precision outside of windows
-  options <- analysisOptions("MixedModelsGLMM")
-  options$Contrasts <- list(list(isContrast = FALSE, levels = c("1", "2", "3", "4", 
+  options <- jaspTools::analysisOptions("MixedModelsGLMM")
```

Comment:
just so you know, it's not mandatory to prefix this with `jaspTools::`, although it's not wrong to add it either.

Reply:
I was actually getting errors when running the tests locally without it